doc/sphinx/addendum/program.rst

```diff
@@ -158,34 +158,33 @@ prove some goals to construct the final definitions.
 Program Definition
 ~~~~~~~~~~~~~~~~~~
 
-.. cmd:: Program Definition @ident := @term
-
-   This command types the value term in Russell and generates proof
+   A :cmd:`Definition` command with the :attr:`program` attribute types
+   the value term in Russell and generates proof
    obligations. Once solved using the commands shown below, it binds the
    final |Coq| term to the name :n:`@ident` in the environment.
 
    .. exn:: @ident already exists.
       :name: @ident already exists. (Program Definition)
       :undocumented:
```

Comment:
Let's remove that. This is redundant with the exact same error message that is already shown for the `Definition` command.